### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by alanshaw**
Updates lotus and `filecoin-ffi` to version currently used by lotus.

Also updates circle config - please check is sane.

### CODE REVIEW

**Review by willscott — APPROVED** [review]
😬

**Comment by willscott** [comment]
are we expecting to need blst operations in our vm replay/generation, or can we stub out that dependency?

**Comment by alanshaw** [comment]
@willscott it would be good to stub...I'll open an issue.

**Review by willscott — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -4,21 +4,15 @@ version: 2.1
 parameters:
   go-version:
     type: string
-    default: "1.14.2"
+    default: "1.14.6"
```
Comment (nonsense) [inline .circleci/config.yml:7]: I think this is not used, now that we use the circleci golang docker image.

Comment (nonsense) [reply .circleci/config.yml:7]: Oh sorry, we still use it as part of the cache dir...

**Pull request merged**